skrf/circuit.py

```diff
@@ -142,6 +142,9 @@ def __init__(self, connections: List[List[Tuple]]) -> None:
             Each connection is a described by a list of tuple.
             Each tuple contains (network, network_port_nb).
             Port number indexing starts from zero.
+            
+        name : string
+            Name assigned to the circuit (network).
```

Comment:
```suggestion
        name : string, optional
            Name assigned to the circuit (Network). Default is None.
```

Reply:
Indeed, like this is more clear. Thank you Julien!